_content/booklets/message-of-the-saints_gunaratna.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "The Message of the Saints: Thera-Therī-Gāthā"
+authors:
+  - gunaratna
+external_url: "https://www.bps.lk/olib/wh/wh135_Gunaratna_Message-of-the-Saints--Thera-Theri-Gatha.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1DVFxp_O5cu9jdlBUteDmqMLeku-mQ-lZ/view?usp=sharing"
+  - "https://drive.google.com/file/d/1QXpZ6KpNGl4vWCG_7S5BPtCAidYYEhQk/view?usp=sharing"
+course: canonical-poetry
+tags:
+  - pali-literature
+  - theravada
+year: 1969
+olid: OL51493374M
+oclc: 33110223
+publisher: bps
+series: wheel
+number: 135
+pages: 39
+---
+A collection of poems from the Thera and Therī Gāthā sections of the Pāli Canon, showcasing the beauty and inspiration of the early Theravada elders, both monks and nuns.
```

Comment:
Description should mention that this anthology contains summaries of the commentarial background stories